addons/hr_holidays/models/hr_leave_allocation.py

```diff
@@ -4,6 +4,7 @@
 from calendar import monthrange
 from datetime import datetime, date, time
 from dateutil.relativedelta import relativedelta
+from odoo.tools import format_date
 
 from odoo import api, fields, models, _
 from odoo.addons.hr_holidays.models.hr_leave import get_employee_from_context
```

Comment:
Keep an eye on the imports order to follow [Guideline](https://www.odoo.com/documentation/18.0/contributing/development/coding_guidelines.html#imports)
```suggestion

from odoo import api, fields, models, _
from odoo.tools import format_date

from odoo.addons.hr_holidays.models.hr_leave import get_employee_from_context
```